packages/broccoli-eyeglass/src/index.ts

```diff
@@ -241,6 +240,28 @@ class EyeglassCompiler extends BroccoliSassCompiler {
     this.buildCache.set(assetImportKey, assetImport);
     return assetImport;
   }
+
+  hashForJs(mod: EyeglassModule): string {
+    let jsPath = mod.mainPath;
+    if (!jsPath) {
+      return ""
+    }
+    let cacheKey = `hashForJs(${jsPath})`;
+    let cachedHash = this.buildCache.get(cacheKey) as string;
+    if (cachedHash) {
+      return cachedHash;
+    }
+    let files = dependencyTree.toList({filename: jsPath, directory: path.dirname(jsPath)});
```

Comment:
As the order of `toList` here will affect the `hash`, do you know it is stable and will not spuriously produce differing hashes. If so awesome, if not you may want to lexicographically sort before hashing.

Reply:
Good question. According to the documentation, this is a topological sort of the dependency order based on how the files are discovered in the source files. My understanding is that it is stable.